deployments/core.go

```diff
@@ -29,6 +27,7 @@ const (
 	coreServiceName         = "fuseml-core"
 	coreServicePort         = 80
 	coreSecretName          = "fuseml-core-gitea"
+	coreConfigMapName       = "config-fuseml-core"
```

Comment:
We already know it's a config, and the name is for a config-map resource type. Does the name really need to reflect that too ? IMO, `fuseml-core` is sufficient.

Reply:
Hm, I took advice from other configMaps we have, and they ofthen (but not always) have this `config-` prefix... (for example under tekton-pipelines and knative-serving namespaces)

Reply:
Ok, I guess it's not such a big deal.

Reply:
I'm just not sure what's the convention. I'm only saying I like it this way little bit more, but I do not _really_ care about it